test/api.test.ts

```diff
@@ -0,0 +1,106 @@
+import assert from 'assert';
+import supertest from 'supertest';
+import app from '../src/app';
+import mongoose from 'mongoose';
+import userModel from '../src/models/userModel';
+
+beforeEach((done) => {
+  mongoose.connect("mongodb://localhost:27017/TestDB",
```

Comment:
make mongoose connection string for tests database configurable via config file or env variables